docs/Prism.html

```diff
@@ -1210,13 +1210,7 @@ <h4 class="name" id=".tokenize"><span class="type-signature">(static) </span>tok
 
     <h5>Example</h5>
     
-    <pre class="prettyprint"><code>let code = `var foo = 0;`;
-let tokens = Prism.tokenize(code, Prism.languages.javascript);
-tokens.forEach(token => {
-    if (token instanceof Prism.Token &amp;&amp; token.type === 'number') {
-        console.log(`Found numeric literal: ${token.content}`);
-    }
-});</code></pre>
+    <pre class="prettyprint"><code>let code = `var foo = 0;`;
let tokens = Prism.tokenize(code, Prism.languages.javascript);
tokens.forEach(token => {
    if (token instanceof Prism.Token &amp;&amp; token.type === 'number') {
        console.log(`Found numeric literal: ${token.content}`);
    }
});</code></pre>
```

Comment:
How? This `^M` means `\r`. How did the line ends of these code blocks specifically get converted to `\r`?

Please revert these changes.

Reply:
That seems to happen when i run the build scripts. I never touched that file by myself. I try to resolve that file.